Makefile

```diff
@@ -12,6 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# It's necessary to set this because some environments don't link sh -> bash.
+SHELL := /bin/bash
```

Comment:
This will cause an issue if the user's system doesn't have bash in /bin, such as Fedora and Mac OSX

Reply:
It might be better to push the logic out into a hack script that uses `#!/usr/bin/env bash` instead.

Reply:
I stole this from https://github.com/kubernetes/kubernetes/blob/master/build/root/Makefile#L33 when I thought my commands were failing due to the shell. It may not be necessary... let me test without it.

Reply:
It works for me without it, so I'm just going to take it out.